README.md

```diff
@@ -56,7 +56,19 @@ nextflow run /path/to/pipeline/ -c /path/to/custom_module.conf
 
 If you decide to upload your module file to `nf-core/modules` then this will ensure that it will be automatically downloaded, and available at run-time to all nf-core pipelines, and to everyone within the Nextflow community! See [`nf-core/modules/nf`](https://github.com/nf-core/modules/tree/master/nf) for examples.
 
-> The definition and standards for module files are still under discussion amongst the community but hopefully, a description should be added here soon!
+The definition and standards for module files are still under discussion amongst the community.
+
+Currently the following points have been agreed on:
+
+- Module file should only define inputs/outputs as parameters and have the ability to use `params.MODULENAME_options` as an additional parameter to add any additional settings via pipelines.
+- Specify single-end boolean values within the input channel and not be inferred from the data e.g. [here](https://github.com/nf-core/tools/blob/028a9b3f9d1ad044e879a1de13d3c3a25a06b9a7/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/modules/nf-core/fastqc.nf#L13)
+- Define threads or resources where required for a particular process using `task.cpus`
+- Software that can be piped together should be added to separate module files unless there is an run-time, storage advantage in implementing in this way e.g. `bwa mem | samtools view` to output BAM instead of SAM
+- Process names should be all uppercase
+- The `publishDirMode` should be configurable
+- Test data is stored within this repo. Re-use generic files from `tests/data` by
```

Comment:
Brilliant! Thanks @grst! Could we also add in here the requirements to have a `environment.yml` and Docker file if the software is available on Conda.